javascript/game.js

```diff
@@ -110,11 +159,28 @@ exports.Game = function () {
     );
   };
 
-  var askQuestion = function () {
-    if (currentCategory() == "Pop") console.log(popQuestions.shift());
-    if (currentCategory() == "Science") console.log(scienceQuestions.shift());
-    if (currentCategory() == "Sports") console.log(sportsQuestions.shift());
-    if (currentCategory() == "Rock") console.log(rockQuestions.shift());
+  this.askQuestion = function (category = currentCategory()) {
+    deckArray.forEach((deck) => {
```

Comment:
If we know the category that the question should be asked for, should we iterate through `deckArray`?